services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -762,12 +826,37 @@ private void setUpIndividualRequest(final IIndividualRequest request) {
         bulkSubmission.setCustomerReference("TEST_CUST_REF");
         bulkSubmission.setId(1L);
         bulkSubmission.setNumberOfRequest(1);
-        final List<IIndividualRequest> requests = new ArrayList<IIndividualRequest>();
+        final List<IIndividualRequest> requests = new ArrayList<>();
         requests.add(request);
 
         bulkSubmission.setIndividualRequests(requests);
 
         request.setBulkSubmission(bulkSubmission);
     }
 
+
+    @Test
+    void testSetIndividualRequestDaoConsumerWriter(){
```

Comment:
The name of the test doesn't appear to match the purpose.  The test appears to be checking that the setIndividualRequestDao() method works but the name includes "ConsumerWriter" which is different.  Rename test so that it matches the purpose.